all.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as alert from '@clayui/alert';
```

Comment:
Pro-Tip™: These can all be:

```js
export * as alert from '@clayui/alert';
```

(May depend on project settings and dependency versions, but I have been able to do this in other TypeScript projects.)

Reply:
For some reason it wouldn't let me name them, so I changed them all to 
```js
export * from '@clayui/package';
```

Reply:
That's quite a bit different though. That won't export the `default` export if you do that; it's just mashing in all the named exports. Note the difference in [how Babel transpiles the two forms](https://babeljs.io/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=KYDwDg9gTgLgBAKjgQwM5xHAZlCBbOAcghgAtgpCBuAKBtElkW1wMLIEsA7Ac2qA&debug=false&forceAllTransforms=true&shippedProposals=true&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Ces2015%2Ces2016%2Ces2017%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3%2Ces2015-loose%2Cenv&prettier=false&targets=&version=7.9.6&externalPlugins=).

> For some reason

I have a feeling it depends on recent enough versions of TypeScript and/or Babel, and possibly also on how they are configured. In liferay-portal, for example, we have Babel [configured](https://github.com/liferay/liferay-npm-tools/blob/b50878235a3bfd187430dea9e12cdca4e03dcfb9/packages/liferay-npm-scripts/src/config/babel.json#L10) to use the `@babel/proposal-export-namespace-from` plugin. (Note: Despite the "proposal" in the name, [this feature made it to Stage 4 and is in the ECMAScript spec at this point](https://github.com/tc39/proposal-export-ns-from)).

Reply:
Strange, the bundle size after switching from `* as` to `* from` didn't change at all. I'll look into adding that babel config and see if I can get it working.